README.md

```diff
@@ -382,24 +382,25 @@ See [the postgres documentation about encoding](https://www.postgresql.org/docs/
 
 ## Postgres Replication Setup
 
-The image supports replication out of the box. The two mains replication methods allowed are
+The image supports replication out of the box. By default replication is turned off.
+The two mains replication methods allowed are
```

Comment:
Yes, I didn't want to run it off for version 12 and below but for version 13 it should befine because people doing an upgrade should ideally check what has changed between successive versions when they upgrade their clusters.